packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -288,12 +288,6 @@ const conf = (module.exports = convict({
       format: Boolean,
       env: 'REACT_CONVERSION_WEB_CHANNEL_EXAMPLE_ROUTES',
     },
-    inlineTotpRoutes: {
-      default: false,
-      doc: 'Enable users to visit the "inline" 2fa setup routes',
-      format: Boolean,
-      env: 'REACT_CONVERSION_INLINE_TOTP_ROUTES',
```

Comment:
We didn't. We made this temporarily since there was an issue with getting to this flow but if it lands this sprint it makes sense to go out with signin IMO.